demisto_sdk/commands/common/tools.py

```diff
@@ -920,6 +920,9 @@ def get_from_version(file_path):
         get_yaml(file_path) if file_path.endswith("yml") else get_json(file_path)
     )
 
+    if not isinstance(data_dictionary, dict):
+        raise ValueError("yml file returned is not of type dict")
```

Comment:
do we not have `yaml`s that are lists?

Reply:
I dont know of such a case and the next line in this function is data_dictionary.get("fromversion") leading to an error.

Reply:
true